priv/repo/migrations/20240416091937_add_email_and_fullname_to_user.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Trento.Repo.Migrations.AddEmailAndFullnameToUser do
```

Comment:
Should the name of the alias not be ```defmodule ... AddEmailAndFullNameToUser do .....``` ?

Reply:
Phoenix generator generated the migration, so we keep like that :eyes: